get-started/what-is-shiny.qmd

```diff
@@ -0,0 +1,170 @@
+---
+title: What is Shiny?
```

Comment:
Suppress edit/report issue links from this page (leave on the rest, tho)
https://github.com/posit-dev/py-shiny-site/pull/261#issuecomment-2822434168